cmd/server/server.go

```diff
@@ -43,12 +47,31 @@ func serverCmd(errorf errorFn) *cobra.Command {
 		Use:   "server",
 		Short: "Starts the mixer as a server",
 		Run: func(cmd *cobra.Command, args []string) {
-			fmt.Printf("Starting gRPC server on port %v", sa.port)
+			fmt.Printf("Starting gRPC server on port %v, type 'exit' to stop gracefully.\n", sa.port)
 
-			err := runServer(sa)
+			// The tracer will keep track of spans in memory and print them to stdout when the server finishes.
```

Comment:
This doesn't seem like an acceptable way to do things in a server that can run indefinitely...

Like in the client, we should make collection of tracing info optional. We probably also should stream the output to a file rather than pile it up in memory.